charts/rancher-aks-operator/1.0.100/templates/clusterrole.yaml

```diff
@@ -0,0 +1,15 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: aks-operator
+  namespace: cattle-system
+rules:
+  - apiGroups: ['']
+    resources: ['secrets']
+    verbs: ['get', 'list', 'create', 'watch']
```

Comment:
Pending changes to aks-operator we may be able to remove some or all of these secrets permissions in the future.